practice-two/src/style/abstracts/variable.css

```diff
@@ -0,0 +1,100 @@
+/* 1 rem is defined as 16px */
+:root{
+  --font-size-base: 16px;
+
+  --font-size-xxsmall: calc(11 / var(--font-size-base) * 1rem);
+  --font-size-xsmall: calc(12 / var(--font-size-base) * 1rem);
+  --font-size-medium: calc(18 / var(--font-size-base) * 1rem);
+  --font-size-large: calc(20 / var(--font-size-base) * 1rem);
+  --font-size-xxxlarge: calc(32 / var(--font-size-base) * 1rem);
+  --font-size-huge: calc(36 / var(--font-size-base) * 1rem);
+  --font-size-xxhuge: calc(64 / var(--font-size-base) * 1rem);
+
+  --line-height-base: 16px;
+
+  --line-height-normal: normal;
+  --line-height-xsmall: calc(13 / var(--line-height-base * 1rem));
```

Comment:
Rem units dictate an element's font size relative to the font size the root element, not relative to line-height